integration_test.go

```diff
@@ -71,6 +105,7 @@ func TestIntegration(t *testing.T) {
 			_, iter, err := engine.Query(session, tt.query)
 			require.NoError(err)
 			rows, err := sql.RowIterToRows(iter)
+			spew.Dump(rows)
```

Comment:
This gives a lot of output. If used it should only print the output on error.

Reply:
Shit, that was a leftover 😞